recipes-core/systemd-conf/files/can0.network

```diff
@@ -0,0 +1,6 @@
+[Match]
+Name=can0
+
+[CAN]
+#BitRate=500K
+#RestartSec=100ms
```

Comment:
Why commented?

Reply:
#Example files for can 0 to 6 (does not do anything but if you remove # for BitRate=500K and RestartSec=100ms it will start can controller(s) on next reboot or use 'networkctl reload' on interface named can0 to 6).

Reply:
written in bb recipe